static/src/javascripts/projects/commercial/modules/cmp/ccpa-ab-test.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+// import { isInUsa } from 'common/modules/commercial/geo-utils';
+import config from 'lib/config';
+
+// let isInTest;
+
+const isInServerSideTest = (): boolean => config.get('tests.ccpaCmp', false);
+
+// export const isInCcpaTest = (): boolean => {
+//     isInTest = isInTest || (isInUsa() && isInServerSideTest());
```

Comment:
if `isInTest` is false it will always run `(isInUsa() && isInServerSideTest())`, how about:

```suggestion
//     if (typeof isInTest === 'undefined') isInTest = isInUsa() && isInServerSideTest();
```